test/test_netmhc_pan.py

```diff
@@ -10,8 +10,23 @@
     "TP53-001": "ASILLLVFYW"
 }
 
+# This test assumes you have versions of netMHCpan installed in your PATH
```

Comment:
Not sure we should assume that people running `mhctools` tests have all versions installed. Maybe make this optional but have it run on Travis?

Reply:
Agree, updated to make the versioned names optional. The test will run those versions if they exist